test/parallel/sdk.API.utils.test.ts

```diff
@@ -213,16 +209,12 @@ test("GIVEN multiSwapSell WHEN operation is confirmed AND isMultiSwapAssetTransa
 });
 
 test("GIVEN multiSwapSell WHEN operation is failed AND isMultiSwapAssetTransactionSuccessful THEN it returns false", async () => {
-  const [token2] = await Assets.setupUserWithCurrencies(
-    sudo,
-    [defaultCurrencyValue],
-    sudo,
-  );
-
-  const tokenIds = [token1, token2];
+  const [testUser2] = setupUsers();
+  await Sudo.batchAsSudoFinalized(Assets.mintNative(testUser2));
+  const tokenIds = [token1, GASP_ASSET_ID];
```

Comment:
ehy?

Reply:
Because if we create multiswap in new pallet for the tokens which don't have own pool we can't create extrinsic: we receive liqId = -1. I changed the error case.